### ISSUE TITLE
Issue with my redirection scheme

### ISSUE THREAD

**Issue by thomasrliberto** [MaikuB/flutter_appauth#127]
Hi,

I am facing the same issue that is discussed [here](https://github.com/MaikuB/flutter_appauth/issues/122). I've read through this thread yet I am still stuck. I am testing this on an Android emulator, and have yet to test it on iOS. Running flutter doctor shows green across the board.

My redirect URI was provisioned on the server as AppName://callback. My app/build.gradle is below:

`defaultConfig {`
      
  `applicationId "com.example.appauthtest1"`

        minSdkVersion 16

        targetSdkVersion 28

        versionCode flutterVersionCode.toInteger()

        versionName flutterVersionName

        manifestPlaceholders = [

                'appAuthRedirectScheme': 'AppName'
 `]`

    `}` 

I apologize if this is a silly question - I have been stuck on the issue for the past two days and I cannot for the life of me figure out why this is occurring. I followed the solution yet I am still stuck. I would appreciate some guidance.

**Comment by MaikuB** [comment]
If scheme matches what you've got as part of the redirect url but you're still running into issues then your problem lies elsewhere and doesn't relate to the plugin so I'll be closing this.

You'll need to check the docs for the identity provider to see if there's something else you're missing that needs to be included and double check you've configured everything else correctly e.g. the redirect url you specify in the code matches what you've registered with the identity provider.

**Comment by thomasrliberto** [comment]
@MaikuB Hi, I know you just closed this issue but I just tried testing the exact same project on an iOS emulator and everything works perfectly, including automatic redirection. I didn't change anything, which makes me think that perhaps there is an actual bug? I know Android Studio just pushed out an update but I can't update it without losing compatibility with the Flutter and Dart packages. I'm new to Mobile dev and IdM so I'm not sure what the issue could be but I wanted to know if you have any input.

**Comment by MaikuB** [comment]
That it works on iOS doesn't mean it'll work on Android. If there was a bug then the same issue should've also occurred in the example app that's in the repository (which you can check as a reference) and there'd be more reports about this issue. The issue you linked to in your original post was due to a mistake by the developer who opened that issue.

As there's not a bug around this, I'd suggest you make use of one of places listed at https://flutter.dev/community such as Stack Overflow or Slack

**Comment by thomasrliberto** [comment]
@MaikuB I finally got it working! Hopefully this will help out anyone in the future with this problem. It turns out all I had to do was set the scheme name for appAuthRedirectScheme to an entirely lower case string. So even though my Redirect URI is AppName:/callback, I had to set appAuthRedirectScheme to appname instead of AppName. This got it working. 

Thanks for your help Michael and thank you for the plugin!
